reddwarf/guestagent/db/models.py

```
@@ -344,23 +344,26 @@ def __init__(self):
         self._databases = []
 
     def _check_valid(self, value):
-        # User names are not valid if they contain unsupported characters, or
-        # are in the ignore_users list.
         if (not value or
             self.not_supported_chars.search(value) or
-            string.find("%r" % value, "\\") != -1 or
-            value.lower() in self._ignore_users):
+            string.find("%r" % value, "\\") != -1):
             return False
         else:
             return True
 
+    def _check_valid_name(self, value):
```

It's originally a Java convention but it seems like it pops up from time to time in Python land. I agree with @ed- that something called "check" I expect to raise an exception rather than return a value.